mlir/python/mlir/_mlir_libs/_mlir/__init__.pyi

```diff
@@ -10,4 +10,4 @@ class _Globals:
     def _check_dialect_module_loaded(self, dialect_namespace: str) -> bool: ...
 
 def register_dialect(dialect_class: type) -> object: ...
-def register_operation(dialect_class: type) -> object: ...
```

Comment:
This is just a drive by change fixing a type error I discovered in `arith.py`.

Reply:
"drive by fix" - I'm gonna use that from now on